### DISCUSSION TITLE
Gradle sync failed: NDK not configured.

### DISCUSSION THREAD

**Discussion by Metadisk** [google/filament#3490] [General]
I downloaded NDK but getting Gradle sync failed: NDK not configured. Download it with SDK manager. Preferred NDK version is '21.1.6352462'. and I gave nkdir in local properties also using android 4.2

**Comment by romainguy** [comment]
We recently switched to NDK `22.0.7026061`, make sure that's the one you have downloaded. You don't need to set `ndkdir` in `local.properties` anymore, AGP takes care of it.

**Reply by Metadisk** [reply]
I removed all old NDK also added environment ANDROID-ROOT-SDK but I am getting this error, I didn't add nkdir in localproperties but I didn't see anything on NDK location

**Reply by Metadisk** [reply]
I have still the same error for samples
